src/components/layout/MunicipalityLayout.tsx

```diff
@@ -104,108 +107,115 @@ function MunicipalityLayout(props: WithChildren<IMunicipalityData>) {
           title="Nederland"
         />
       </Head>
-
-      <div className="municipality-layout">
-        {showAside && (
-          <aside className="municipality-aside">
-            <Combobox<IMunicipality>
-              handleSelect={handleMunicipalitySelect}
-              options={municipalities}
-            />
-
-            {showMetricLinks && (
-              <nav aria-label="metric navigation">
-                <h2>{siteText.nationaal_layout.headings.medisch}</h2>
-                <ul>
-                  <li>
-                    <Link
-                      href="/gemeente/[code]/positief-geteste-mensen"
-                      as={`/gemeente/${code}/positief-geteste-mensen`}
+      <div
+        className={`municipality-layout  small-screen-${displayTendency}-tendency`}
+      >
+        {!isMainRoute && (
+          <Link href="/gemeente/[code]" as={`/gemeente/${code}`}>
+            <a className="back-button" href={`/gemeente/${code}`}>
```

Comment:
`ahref` should only exist on `Link`
```suggestion
            <a className="back-button">
```